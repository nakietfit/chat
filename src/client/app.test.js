import App from './app'
import React from 'react'
import { shallow } from 'enzyme'

describe('App', () => {
  test('App renders nested components', () => {
    const app = shallow(<App />);
    expect(app.find('Chat').length).toEqual(1);
  });
})