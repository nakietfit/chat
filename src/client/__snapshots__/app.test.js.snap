// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App renders nested components 1`] = `ShallowWrapper {}`;

exports[`App should match snapshot 1`] = `ShallowWrapper {}`;
